client/web/src/enterprise/insights/pages/insights/creation/capture-group/components/query-input/CaptureGroupQueryInput.tsx

```diff
@@ -16,11 +16,10 @@ export interface CaptureGroupQueryInputProps extends Omit<InsightQueryInputProps
 
 export const CaptureGroupQueryInput = forwardRef<HTMLInputElement, CaptureGroupQueryInputProps>((props, reference) => (
     <InsightQueryInput {...props} ref={reference} patternType={SearchPatternType.regexp}>
-        <Button variant="icon" className={styles.regexButton} disabled={true}>
-            <RegexIcon
-                size={21}
-                data-tooltip="Regular expression is the only pattern type usable with capture groups and it’s enabled by default for this search input."
-            />
-        </Button>
+        <Tooltip content="Regular expression is the only pattern type usable with capture groups and it’s enabled by default for this search input.">
+            <Button variant="icon" className={styles.regexButton} disabled={true}>
+                <Icon svgPath={mdiRegex} size="md" aria-hidden={true} />
```

Comment:
```suggestion
                <Icon inline={false} height={21} width={21} svgPath={mdiRegex} aria-hidden={true} />
```

Unsure that `size="md"` will ensure no visual regressions